jenkins/catapult/environment.py

```diff
@@ -71,7 +71,8 @@ def rmtree(self, path):
 		if self.dry_run:
 			return
 
-		shutil.rmtree(path, onerror=rm_failure_handler)
+		kwargs = {'onexc' if sys.version_info >= (3, 12) else 'onerror': rm_failure_handler}
+		shutil.rmtree(path, **kwargs)
```

Comment:
i believe callback parameters changed, so callback needs updating too
https://docs.python.org/3/library/shutil.html#shutil.rmtree

Reply:
yea, it seems the third parameter has change